documentation/wiki/ChangeWaves.md

```diff
@@ -1,15 +1,16 @@
 # What are Change Waves?
-A Change Wave is a set of risky features developed under the same opt-out flag. This flag happens to be the version of MSBuild that the features were developed for. The purpose of this is to warn developers of risky changes that will become standard functionality down the line. 
+A Change Wave is a version in which a set of risky features exists under the same opt-out flag. This flag happens to be the version of MSBuild that the features were developed for. The purpose of this is to warn developers of risky changes that will become standard functionality down the line. 
 
 ## How do they work?
-The opt out comes in the form of setting the environment variable `MSBuildDisableFeaturesFromVersion` to the wave (or version) that contains the feature you want **disabled**. See the mapping of change waves to features below.
+The opt out comes in the form of setting the environment variable `MSBuildDisableFeaturesFromVersion` to the Change Wave (or version) that contains the feature you want **disabled**. See the mapping of change waves to features below.
 
-**Note:** If  `MSBuildDisableFeaturesFromVersion` is set to `16.8`, this will **disable** all features under that `16.8` and **any further versions**.
-
-## MSBuildDisableFeaturesFromVersion Values
-- If `MSBuildDisableFeaturesFromVersion` is not set, all change waves will **be enabled**
-- If `MSBuildDisableFeaturesFromVersion` is set to some out of bounds version (see current rotation of waves below), you will be defaulted to the lowest wave. This will **disable all waves**.
-- If `MSBuildDisableFeaturesFromVersion` is set to some invalid format (ex: 16x8, 17_0), all change waves will **be enabled**.
+## MSBuildDisableFeaturesFromVersion Values & Outcomes
+| `MSBuildDisableFeaturesFromVersion` Value                         | Result        | Receive Warning? |
+| :-------------                                                    | :----------   | :----------: |
+| Unset                                                             | All Change Waves will be enabled, meaning all features behind each Change Wave will be enabled.               | No   |
+| `16.8` (or any valid & current Change Wave)                       | All features behind the `16.8` and above Change Wave will be disabled    .                                    | No   |
+| Invalid Value (Invalid Change Wave, or out-of-rotation version)   | You will be defaulted to the closest valid value (ascending). Ex: Setting `16.9` will default you to `16.10`. | Yes  |
```

Comment:
Is this right? I thought we changed this.

Reply:
The only thing that was changed here was if `MSBuildDisableFeaturesFromVersion` was set to something higher than the highest wave, it would clamp down to the highest wave (before, it clamped to the lowest wave). Hopefully the wording here implies that.

Reply:
I should have been more precise—I think the message is right (and clear), but I thought we said no warning?

Reply:
Oh you're right, there's actually two cases here. Out of rotation **AND** invalid value. One receives a warning and the other does not. I'll clarify. Thanks!